utils/build.ps1

```diff
@@ -1728,7 +1723,6 @@ function Build-Dispatch([Platform]$Platform, $Arch, [switch]$Test = $false) {
     -Arch $Arch `
     -Platform $Platform `
     -UseBuiltCompilers C,CXX,Swift `
```

Comment:
Is it okay not to keep `ExperimentalTest` when testing? If `$Target` is really dead, better to remove it from the above statement?

Reply:
No, that needs to be preserved to run the tests.

Reply:
So, we need to keep the `-BuildTargets $Targets` here?